File: chatbot.js
function sendMessage(){

    let input = document.getElementById("userInput");
    let message = input.value.trim();

    if(message === "") return;

    let chatBox = document.getElementById("chatBox");

    chatBox.innerHTML += `<div class="user">${message}</div>`;

    let reply = getAIResponse(message);

    setTimeout(()=>{
        chatBox.innerHTML += `<div class="bot">${reply}</div>`;
        chatBox.scrollTop = chatBox.scrollHeight;
    },500);

    input.value="";
}

function getAIResponse(msg){

    msg = msg.toLowerCase();

    if(msg.includes("fever"))
        return "Take rest and drink fluids.";

    if(msg.includes("headache"))
        return "Relax and stay hydrated.";

    if(msg.includes("hello"))
        return "Hello! I am your AI Health Assistant.";

    return "Please consult a doctor for professional advice.";
}

File: chatbot.py
import pandas as pd

data = pd.read_csv("model/health_data.csv")

def get_ai_response(message):

    msg = message.lower()

    for i,row in data.iterrows():
        if row["symptom"] in msg:
            return f"Possible condition: {row['disease']}. Advice: {row['advice']}"

    return "Please consult a healthcare professional."
